share/workspaces/CMakeLists.txt

```diff
@@ -19,10 +19,10 @@
 
 function (build_workspace WS_NAME _TARGET)
       set(_OUTPUT "${CMAKE_CURRENT_BINARY_DIR}/${WS_NAME}.workspace")
-      set(_FILES workspace.xml META-INF/container.xml)
+      set(_FILES ${WS_NAME}.xml META-INF/container.xml)
       add_custom_command(OUTPUT ${_OUTPUT}
             COMMAND ${CMAKE_COMMAND} -E tar cf "${_OUTPUT}" --format=zip -- ${_FILES}
-            DEPENDS
+            DEPENDS _FILES
```

Comment:
I don't think `DEPENDS` can take list arguments, so this code effectively depends on a files called "_FILES". You need to use `${_FILES}` rather than `_FILES` to force the list to be expanded.

Reply:
Oops, will fix, after having had a walk to catch some fresh air

Reply:
Don't bother. I just tried on my branch and confirmed some other changes are needed before it will work as intended. I will fix those in another PR and then you'll be have to rebase, but it will be a trivial rebase.

Reply:
I did it anyway, but indeed there's something severly wrong, Travis doesn't build this PR

Reply:
actually I removed that change...